README.md

```diff
@@ -63,7 +63,7 @@ Teamware is developed by the [GATE](https://gate.ac.uk) team, an academic resear
 ## Citation
 For published work that has used Teamware, please cite this repository. One way is to include a citation such as:
 
-> Karmakharm, T., Wilby, D., Roberts, I., & Bontcheva, K. (2022). GATE Teamware (Version 0.1.4) [Computer software]. https://github.com/GateNLP/gate-teamware
+> Karmakharm, T., Wilby, D., Roberts, I., & Bontcheva, K. (2022). GATE Teamware (Version 2.1.0) [Computer software]. https://github.com/GateNLP/gate-teamware
```

Comment:
Should we add something in the version.py to update the version number here or at least a check as a reminder?

Reply:
Added this to `version.py` in [1ab0cf2](https://github.com/GateNLP/gate-teamware/pull/364/commits/1ab0cf2d9ff53bddb0b548c7ca78f1d478874c08)